pywt/tests/test_multilevel.py

```diff
@@ -50,6 +52,56 @@ def test_swt_decomposition():
     assert_(pywt.swt_max_level(len(x)) == 3)
 
 
+def test_swt_iswt_integration():
+    """
```

Comment:
You don't want to add docstrings to test functions/methods, because then those are what gets displayed when running ``test(verbose=2)`` instead of ``test_file.test_name``. So just changing this string to a multi-line comment is the way to go.